src/meshReaders/Hex_Mesh_Reader.cc

```diff
@@ -217,23 +219,19 @@ Hex_Mesh_Reader::get_unique_element_types() const {
 /*!
  * There is no provision for naming element sets in the Hex format. The
  * following default names are provided for the sets found on the mesh file:
- * <ul>
- *   <li> "Interior" -- All the ndim-dimensional cells in the problem.
- *   <li> "Interior_Region_x" - Interior cells with integer flag "x".
- *   <li> "Vacumm_Boundary" -- All the (ndim-1)dimensional vacuum boundary
-*         faces.
- *   <li> "Vacuum_Boundary_Region_x" -- Vacuum boundary faces with
- *        flag integer "x"
- *   <li> "Reflective_Boundary" -- All the (ndim-1) dimensional reflective
- *        boundary faces.
- * </ul>
+ * - "Interior" -- All the ndim-dimensional cells in the problem.
+ * - "Interior_Region_x" - Interior cells with integer flag "x".
+ * - "Vacumm_Boundary" -- All the (ndim-1)dimensional vacuum boundary faces.
+ * - "Vacuum_Boundary_Region_x" -- Vacuum boundary faces with flag integer "x"
+ * - "Reflective_Boundary" -- All the (ndim-1) dimensional reflective boundary
+ *   faces.
  */
 std::map<std::string, std::set<unsigned>>
 Hex_Mesh_Reader::get_element_sets() const {
   // Alternatively, the private data of the class could be changed so that the
-  // work done here is done in the constructor. This would be more efficient
-  // if this is going to be used repetively.
-  typedef std::map<std::string, std::set<unsigned>> resultT;
+  // work done here is done in the constructor. This would be more efficient if
+  // this is going to be used repetively.
```

Comment:
This might be outside scope, but "repetitively" is misspelled :)

Reply:
You've seem my other PRs; I'm all about fixing spelling errors.  MSVC has a built in spell checker that works like Word.  It underlines the misspelled words.